recipes/carpedeam/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "1.0.0" %}
+{% set sha256 = "92deec3d670810f8433c0345758588b47fa460349e61ce8aa549374f5cf4a812" %}
+
+package:
+  name: carpedeam 
+  version: {{ version|replace("-", ".")  }}
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('carpedeam', max_pin=None) }}
+
+source:
+  url: https://github.com/LouisPwr/CarpeDeam/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+    - make
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  host:
+    - perl
+    - gawk
+    - zlib
+    - bzip2
+  run:
+    - gawk
+    - zlib
+    - bzip2
+
+test:
+  commands:
+    - carpedeam > /dev/null
```

Comment:
Such kind of tests usually pipe to `grep`, e.g. `carpedeam --version | grep {{ version }}` or `carpedeam --help | grep "Some expected text"`. If the command returns non-0 status code then it is considered an error.

Reply:
That is good to know! Thanks. I added a more informative test.